lib/hpc/cluster.pm

```diff
@@ -1,24 +1,21 @@
 # SUSE's openQA tests
 #
-# Copyright 2020 SUSE LLC
+# Copyright 2020-2022 SUSE LLC
```

Comment:
don't update copyright years with ranges, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md?plain=1#L49

Reply:
*this change is ready for review*